cluster/gce/gci/configure-helper.sh

```diff
@@ -990,6 +995,7 @@ function start-kube-apiserver {
   sed -i -e "s@{{admission_controller_config_volume}}@${admission_controller_config_volume}@g" "${src_file}"
   sed -i -e "s@{{image_policy_webhook_config_mount}}@${image_policy_webhook_config_mount}@g" "${src_file}"
   sed -i -e "s@{{image_policy_webhook_config_volume}}@${image_policy_webhook_config_volume}@g" "${src_file}"
+  sed -i -e "s@{{pillar\['enable_cache_mutation_detector'\]}}@${ENABLE_CACHE_MUTATION_DETECTOR}@g" "${src_file}"
```

Comment:
One of the failing jobs couldn't get all of the system pods running (e.g. `event for fluentd-gcp-v1.38-tf9jx: {kubelet e2e-gce-agent-pr-43-0-minion-group-wm8d} FailedSync: Error syncing pod, skipping: network is not ready: [runtime network not ready: NetworkReady=false reason:NetworkPluginNotReady message:docker: network plugin is not ready: Kubenet does not have netConfig. This is most likely due to lack of PodCIDR]`)